src/pretix/control/forms/vouchers.py

```diff
@@ -224,10 +227,17 @@ def clean(self):
                       'them.')
                 ]
             })
-        Voucher.clean_subevent(
-            data, self.instance.event
-        )
-        Voucher.clean_max_usages(data, self.instance.redeemed)
+
+        try:
+            Voucher.clean_subevent(
+                data, self.instance.event
+            )
+        except ValidationError as e:
+            raise ValidationError({"quota": e.message})
```

Comment:
`clean_subevent` checks if block_quota is set, but the error still says 'If you want this voucher to block quota, you need to select a specific date.' – so IMHO we should assign the error to `subevent`.

```suggestion
            raise ValidationError({"subevent": e.message})
```

Also, if you still want to assign to quota, I guess you meant `block_quota` as quota is not a field on the form and fails assigning the error.